### PR TITLE
Add interop network

### PR DESCRIPTION
**Pull request by travisperson**

### CODE REVIEW

File: build/params_interop.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+func init() {
+	policy.SetSupportedProofTypes(
+		abi.RegisteredSealProof_StackedDrg2KiBV1,
+		abi.RegisteredSealProof_StackedDrg8MiBV1,
+		abi.RegisteredSealProof_StackedDrg512MiBV1,
+	)
+	policy.SetConsensusMinerMinPower(abi.NewStoragePower(2048))
+	policy.SetMinVerifiedDealSize(abi.NewStoragePower(256))
+	policy.SetPreCommitChallengeDelay(abi.ChainEpoch(10))
```
Comment (travisperson) [inline build/params_interop.go:59]: Should these be any special values? These were pulled from the 2k network params.

File: build/params_interop.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	UpgradeLiftoffHeight = getUpgradeHeight("LOTUS_LIFTOFF_HEIGHT", UpgradeLiftoffHeight)
+	UpgradeKumquatHeight = getUpgradeHeight("LOTUS_KUMQUAT_HEIGHT", UpgradeKumquatHeight)
+	UpgradeCalicoHeight = getUpgradeHeight("LOTUS_CALICO_HEIGHT", UpgradeCalicoHeight)
+	UpgradePersianHeight = getUpgradeHeight("LOTUS_PERSIAN_HEIGHT", UpgradePersianHeight)
+	UpgradeOrangeHeight = getUpgradeHeight("LOTUS_ORANGE_HEIGHT", UpgradeOrangeHeight)
+	UpgradeClausHeight = getUpgradeHeight("LOTUS_CLAUS_HEIGHT", UpgradeClausHeight)
+	UpgradeTrustHeight = getUpgradeHeight("LOTUS_ACTORSV3_HEIGHT", UpgradeTrustHeight)
+	UpgradeNorwegianHeight = getUpgradeHeight("LOTUS_NORWEGIAN_HEIGHT", UpgradeNorwegianHeight)
+	UpgradeTurboHeight = getUpgradeHeight("LOTUS_ACTORSV4_HEIGHT", UpgradeTurboHeight)
+	UpgradeHyperdriveHeight = getUpgradeHeight("LOTUS_HYPER_DRIVE_HEIGHT", UpgradeHyperdriveHeight)
```
Comment (travisperson) [inline build/params_interop.go:90]: One word? `LOTUS_HYPERDRIVE_HEIGHT`

File: build/params_interop.go (hunk 1 above)
Comment (arajasek) [reply build/params_interop.go:59]: Should be fine.

**Review by arajasek — APPROVED** [review]
LGTM

**Review by magik6k — APPROVED** [review]

**Pull request merged**